x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/connector/api_key_configuration.tsx

```diff
@@ -96,17 +98,29 @@ export const ApiKeyConfig: React.FC<{ hasApiKey: boolean; indexName: string }> =
       {isModalVisible && <ConfirmModal onCancel={onCancel} onConfirm={onConfirm} />}
       <EuiFlexItem>
         <EuiText size="s">
-          {i18n.translate(
-            'xpack.enterpriseSearch.content.indices.configurationConnector.apiKey.description',
-            {
-              defaultMessage:
-                'First, generate an Elasticsearch API key. This {apiKeyName} key will enable read and write permissions for the connector to index documents to the created {indexName} index. Save the key in a safe place, as you will need it to configure your connector.',
-              values: {
-                apiKeyName: `${indexName}-connector`,
-                indexName,
-              },
-            }
-          )}
+          {!isNative
```

Comment:
nitpick: conditionals are easier to understand if they're positive rather than negative checks. So I'd rather see `isNative ? {nativething} : {nonnativething}`.